fontforge/lookups.c

```diff
@@ -25,7 +25,12 @@
  * OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
  * ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+
+#include "lookups.h"
```

Comment:
`lookups.h` contains the prototype declarations for the functions implemented in `lookups.c`. It should be the very first file to be included so that we are sure that it is self-contained, i.e. includes all the headers that contains the datatypes used in those prototype declarations. This allows us to make sure that the compilation of any files does depend on the order in which other headers are included.